<launch>
 <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true">
    <rosparam file="$(find turtlebot3_dqn)/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find turtlebot3_dqn)/config/costmap_common_params.yaml" command="load" />
 </node>
</launch>

